src/components/result-view-tab.js

```diff
@@ -52,10 +53,16 @@ export const ResultViewTab = ({
 
     const intl = useIntl();
 
+    const loadflowNotif = useSelector((state) => state.loadflowNotif);
+
     function renderLoadFlowResult() {
         return (
             <Paper className={classes.table}>
-                <LoadFlowResult result={loadFlowInfos?.loadFlowResult} />
+                <LoadFlowResult
+                    result={
+                        loadflowNotif ? loadFlowInfos?.loadFlowResult : null
```

Comment:
You don't need to use the ternary when you use the optional chaining.
-> loadFlowInfos?.loadFlowResult ?? null
if you just want to avoid undefined.

Reply:
This is to avoid having results when there is no notification and it is also to empty the table of results when the calculation is invalidated when you log or trigger a line

Reply:
it's fixed